app/assets/js/html5lightbox/icons/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+Font license info
```

Comment:
This file isn't needed in project. I would remove it.

@Update
Now I see that this library have many additional things, so one file will not make a big difference. But again.. Better way would be to use bower for such library